content/influxdb3/core/reference/influxDB3 internals /data durability/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: InfluxDB 3 Data durability
+list_title: Data durability
+description: >
+  The `influxdb3` CLI runs and interacts with the InfluxDB 3 Core server.
```

Comment:
This description doesn't fit the page--seems like it's for the `influxdb3` CLI reference.